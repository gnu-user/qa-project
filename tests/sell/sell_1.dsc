TITLE
    Sell Transaction Privileges (buy-standard)
DESCRIPTION
    Run the login transaction using a buy-standard account. Initiate the sell 
    transaction. Verify that the transaction is rejected and an error occurs.
DEPENDENCIES
