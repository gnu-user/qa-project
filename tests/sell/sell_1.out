Welcome.
Current user accounts file read successfully.
Available tickets file read successfully.
> 
Enter username: 
Username accepted.
> 
Invalid transaction, you do not have the necessary privileges.
> 